src/ui/pages/DeckBuilder/DeckBuilder.tsx

```diff
@@ -0,0 +1,5 @@
+import { Typography } from '@mui/material'
+
+export const DeckBuilder = () => {
+  return <Typography variant="h1">Deck Builder</Typography>
+}
```

Comment:
This needs to render the `DeckBuilder` component from `src/ui/components/DeckBuilder/DeckBuilder.tsx`.